cli/hub/repositories.go

```diff
@@ -0,0 +1,107 @@
+package hub
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+)
+
+const (
+	// repositoriesURL path to the Hub API listing the repositories
+	repositoriesURL = "/v2/repositories/%s/"
+)
+
+// Repository represents a Docker Hub repository
+type Repository struct {
```

Comment:
If it's just to test, maybe we don't need that and we can only check if the response from Hub is OK (no error and json). We don't need to validate the response is a real repositories one.